spec/features/allocation_information_spec.rb

```diff
@@ -42,6 +43,54 @@
     end
   end
 
+  context 'when an NPS, Determinate, English  offender has a key worker assigned' do
+    context 'when the offender has over 10 months left to serve' do
+      before do
+        create_case_information_for(offender_no)
+        create_allocation(offender_no)
+
+        stub_api_calls_for_prison_allocation_path(sentence_start_date: "2020-01-01",
+                                                  conditional_release_date: "2020-11-02",
+                                                  automatic_release_date: "2020-11-02",
+                                                  hdced: "2020-11-02",
+                                                  release_date: "2023-01-01")
+
+        visit prison_allocation_path('LEI', nomis_offender_id: offender_no)
+      end
+
+      it "displays a POM as responsible" do
+        expect(page).to have_content('Responsible')
```

Comment:
Are there multiple people displayed on this page? Is this test specific enough?

Reply:
No, only a single person. It would show up as either supporting or responsible.